<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectCorrespondences</class>
 <widget class="QWidget" name="SelectCorrespondences">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1168</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">SelectCorrespondences</string>
  </property>
  <layout class="QVBoxLayout" name="vl_base">
   <item>
    <widget class="QSplitter" name="splitter_base">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget_base">
      <layout class="QVBoxLayout" name="vl_operations">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>25</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>25</number>
       </property>
       <item>
        <widget class="QPushButton" name="btn_newSession">
         <property name="text">
          <string>New Session</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_refreshSessions">
         <property name="text">
          <string>Refesh Sessions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_sessionList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_refreshOpertations">
         <property name="text">
          <string>Refresh Operations</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_operationList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_newTagOperation">
         <property name="text">
          <string>New Tag Operation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_deleteCurrentOperation">
         <property name="text">
          <string>Delete Current Operation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_resetTagOperation">
         <property name="text">
          <string>Reset Current Tag Operation</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_getInitialPair">
         <property name="text">
          <string>Get Initial Pair</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_showTaggedPointsTable">
         <property name="text">
          <string notr="true">Show Tagged Points Table</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_sendTagFeatures">
         <property name="text">
          <string>Send current tagged</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_mergeCurrentTagged">
         <property name="text">
          <string>Merge Current Tagged</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>  Calculate Operations</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_sfm_command"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_kickoutCalculation">
         <property name="text">
          <string notr="true">Kickout Calculation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_reloadSFMResult">
         <property name="text">
          <string>Reload SFM Result</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_close">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string notr="true">Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_scene">
      <layout class="QVBoxLayout" name="vl_operationscenes" stretch="5,80,5">
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string notr="true">Image Files:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lE_imageFilesDir">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_browseImageDir">
           <property name="text">
            <string notr="true">Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSplitter" name="splitter_scenes">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="opaqueResize">
          <bool>true</bool>
         </property>
         <property name="handleWidth">
          <number>8</number>
         </property>
         <property name="childrenCollapsible">
          <bool>false</bool>
         </property>
         <widget class="QGroupBox" name="GB_ImageView">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string notr="true">Select Image Points</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="VB_imageView" stretch="3,3,94">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <property name="sizeConstraint">
              <enum>QLayout::SetMinimumSize</enum>
             </property>
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="spacing">
              <number>2</number>
             </property>
             <item row="0" column="0" alignment="Qt::AlignTop">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Shift + LMB:  Add new point tag</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Shift + RMB:  Remove selected tag</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0" alignment="Qt::AlignTop">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Mouse Wheel:  Zoom view</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1" alignment="Qt::AlignTop">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Alt + LMB:  Pane view          </string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="5,15,80,0">
             <property name="spacing">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string notr="true">Frame Offset:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lE_FrameOffet">
               <property name="inputMask">
                <string notr="true"/>
               </property>
               <property name="text">
                <string notr="true">0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lb_imageFileName">
               <property name="text">
                <string notr="true">File Name:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lb_imagePosition">
               <property name="text">
                <string>x: y: </string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="vL_ImageView"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="GB_3DView">
          <property name="title">
           <string notr="true">Select 3D Points</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="VB_3DView" stretch="10,90">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>15</number>
             </property>
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>5</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="QPushButton" name="btn_startSelectTagPoints">
               <property name="text">
                <string notr="true">  Selecting Mode  </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_endSelectTagPoints">
               <property name="text">
                <string notr="true">  End Selecting  </string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QFrame" name="frm_3DView">
             <property name="focusPolicy">
              <enum>Qt::TabFocus</enum>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="vl_3DView">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="80,5,5,5,10">
         <item>
          <widget class="QSlider" name="hS_frame">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_previousFrame">
           <property name="text">
            <string notr="true">Previous</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_nextFrame">
           <property name="text">
            <string notr="true">Next</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string notr="true">Frame:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sB_frame"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
